src/reducers/language.js

```diff
@@ -0,0 +1,19 @@
+export default function(state='norway', action) {
+  switch (action.type) {
+    case 'SET_LANGUAGE':
+      return isLanguageValid(action.payload) ? action.payload : null;
+  }
+  return state;
+}
+
+function isLanguageValid(language) {
+  const languages = [
+    'norway',
+    'norway-nynorsk',
+    'sweden',
+    'denmark',
+    'england'
```

Comment:
Burde language være f.eks "norwegian, swedish, english" osv., isdetedet for navnet på landet?